### PR TITLE
Activity Panel: Inbox: Add Store, Enable Inbox

### PR DESCRIPTION
**Pull request by allendav**
See #176 

This builds on the read-only REST API work in https://github.com/woocommerce/wc-admin/pull/507 to actually connect the inbox items (read only for now).

To test
Create a few notes if you haven't already using the https://github.com/Automattic/wc-admin-notes-test-bench/
In wp-admin > WooCommerce note the Inbox is back. Click on it.
Observe the notes.
Profit!

### CODE REVIEW

**Review by timmyc — COMMENTED** [review]
Super awesome to see this working!

image

Couple of quick comments on the code - all of which are fine for follow-up PRs too. Also would be nice to add in some tests around the wp.data goods in a follow-up PR.

File: client/store/notes/resolvers.js
Comment (allendav) [reply client/store/notes/resolvers.js]: LOL. Probably not. Copied without carefully reading it :) Let's keep this for now and bump this into an error handling/display issue?

File: client/header/activity-panel/panels/inbox.js
Comment (allendav) [reply client/header/activity-panel/panels/inbox.js]: Good idea. Will do.

Comment (allendav) [reply client/header/activity-panel/panels/inbox.js]: Actually, I can add tests without breaking this into a separate component. Will do so.

File: client/header/activity-panel/panels/inbox.js
Comment (allendav) [reply client/header/activity-panel/panels/inbox.js]: Exactly - I wanted to make it clear at this point that these aren't clickable / don't do anything - to avoid giving the wrong impression.

File: client/header/activity-panel/index.js
Comment (allendav) [reply client/header/activity-panel/index.js]: We have a task for this. I'll make it an issue.

Comment (allendav) [reply client/header/activity-panel/index.js]: https://github.com/woocommerce/wc-admin/issues/545

File: client/store/notes/resolvers.js
Comment (allendav) [reply client/store/notes/resolvers.js]: This was copied from our old approach in the orders actions. I like what @Aljullu did there recently and will replace this with that approach.

**Review by timmyc — APPROVED** [review]
Thanks for the responses, I'm good with this getting merged in anytime 🤖

File: client/store/notes/resolvers.js
Comment (allendav) [reply client/store/notes/resolvers.js]: Fixed in https://github.com/woocommerce/wc-admin/pull/513/commits/c84dc4879e98ea2e710033480974722ecdaabb78

Will hook up error visually next

Comment (allendav) [reply client/store/notes/resolvers.js]: Error now hooked up for end user too. https://github.com/woocommerce/wc-admin/pull/513/commits/57f8a60f675b4fdca3add8b843469d5ebb5b507f 

Here is what it looks like when it happens:

screen shot 2018-10-16 at 3 29 18 pm

File: client/header/activity-panel/panels/inbox.js
Comment (allendav) [reply client/header/activity-panel/panels/inbox.js]: Fixed in https://github.com/woocommerce/wc-admin/pull/513/commits/a4e0afa50a9309e60278222ff6671316275ab3ed

**Comment by allendav** [comment]
@timmyc - all feedback is in - will merge after I merge #507 which should be tomorrow

**Pull request merged**
